src/main/resources/reference.conf

```diff
@@ -98,13 +97,10 @@ akka-persistence-jdbc {
         # ensures that the database does not get dropped while we are using it
         keepAliveConnection = on
 
-        # 5 * number of cores
+        # See some tips on thread/connection pool sizing on https://github.com/brettwooldridge/HikariCP/wiki/About-Pool-Sizing
+        # Keep in mind that the number of threads must equal the maximum number of connections.
```

Comment:
we should eventually deprecate it and only have `maxConnections`. Not something to do right now, though